#CU-f8wf8d Add necessary libraries assets & folders

brod26 opened the pull request:
**ClickUp Ticket:** https://app.clickup.com/t/f8wf8d

### Ticket Items To Review
- [ ] AC's & Raids from tickets above
- [ ] `Additional review items here...`

### Quick Start
```
1. Open a6 games github repo
2. New folders created: assets, _sass, _layouts
3. In terminal, navigate to a6 repo and run "bundle exec jekyll serve"
```
### Media Resources
website page will look like this
![image](https://user-images.githubusercontent.com/54254425/98860478-96a2d180-2418-11eb-9472-c8c581a32f60.png)


### Additional Notes 
- Add/Remove Top Level Folders & Files
- Setup default html as default layout html page
- Edit _config.yml to ensure it works with default.html

brod26 committed:
Add necessary libraries assets & folders 

-Add/Remove Top Level Folders & Files
-Setup default html as default layout html page
-Edit _config.yml to ensure it works with default.html

thsbrown commented on _layouts/default.html:
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title> A6 Games | A6 Games</title>
+    {% feed_meta %}
+    {% seo %}
+</head> 
+<body>
+    <h1>A6 Games</h1>
+</body>
```
So in general in template languages like HMTL / XML we indent inner tags in order to ensure we can quickly see which tags are contained in other ones. Also it looks like you may have already known that and this maybe was just a an oversight so sorry if I'm sounding patronizing 🤣. I know you said you were learning HTML so just wanted to clarify just in case.

So let's go ahead and indent everything inside the `html` tags 👍.

thsbrown reviewed (changes requested):
Overall looks pretty good bro. I noticed there was a couple AC's that were either missed or the commit / changes didn't go through 👍. 

I went ahead and updated the AC's in the ticket to reflect the AC's that still need work ([Ticket](https://app.clickup.com/t/f8wf8d). Also here's an image for convenience.

![image](https://user-images.githubusercontent.com/2967721/99112597-24aec180-25ab-11eb-92d2-457705d7838b.png).

Also formatting of the PR looks pretty good 👍. The only issue I noticed was the title of the pr was a tad off. I know that's a pain in the ass to remember how to format though and really should have some sort of templating.

It should be #CU-f8wf8d feat some description here.

brod26 committed:
PR Edits

-Add CSS, JS, & IMG folders to proper directory
-Delete files in _posts folder
-Create styles.css and add snippet 3

brod26 commented:
Okay I created the three top level folders again, swear I did had done this already lol, maybe I didn't stage it right?

thsbrown commented:
Yeah it could have been that the rename & delete didn't get flagged 🤔 .

Ah sorry one more change on this one. If you can just indent appropriately in `default.html` & change pr title. See comment [here](https://github.com/A6-Games/a6-games.github.io/pull/2#discussion_r523181622) & bottom of comment [here](https://github.com/A6-Games/a6-games.github.io/pull/2#pullrequestreview-530352136)

thsbrown reviewed (changes requested):
Ah it looks like you also moved the img, js, & css folders outside of the assets folder btw. Those should be contained within assets folder.

brod26 committed:
PR Edits

-css, img, and js folders moved under assets
-proper indents on html 
-changed story name

thsbrown reviewed (approved):
Looks good!
